tests/utils/test_video_utils.py

```diff
@@ -0,0 +1,286 @@
+        self.assertTrue(np.array_equal(videos_list[0], video))
+
+        # Test a list of videos is converted to a list of videos
+        video = get_random_video(16, 32)
+        videos = [video, video]
+        videos_list = make_batched_videos(videos)
+        self.assertIsInstance(videos_list, list)
+        self.assertIsInstance(videos_list[0], np.ndarray)
+        self.assertEqual(videos_list[0].shape, (8, 16, 32, 3))
+        self.assertTrue(np.array_equal(videos_list[0], video))
+
+    @require_torch
+    def test_make_batched_videos_torch(self):
+        # Test a single image is converted to a list of 1 video with 1 frame
+        video = get_random_video(16, 32)[0]
+        torch_video = torch.from_numpy(video)
+        videos_list = make_batched_videos(torch_video)
+        self.assertIsInstance(videos_list, list)
+        self.assertIsInstance(videos_list[0], np.ndarray)
+        self.assertEqual(videos_list[0].shape, (1, 16, 32, 3))
+        self.assertTrue(np.array_equal(videos_list[0][0], video))
+
+        # Test a 4d array of videos is converted to a a list of 1 video
+        video = get_random_video(16, 32)
+        torch_video = torch.from_numpy(video)
+        videos_list = make_batched_videos(torch_video)
+        self.assertIsInstance(videos_list, list)
+        self.assertIsInstance(videos_list[0], torch.Tensor)
+        self.assertEqual(videos_list[0].shape, (8, 16, 32, 3))
+        self.assertTrue(np.array_equal(videos_list[0], video))
+
+        # Test a list of videos is converted to a list of videos
+        video = get_random_video(16, 32)
+        torch_video = torch.from_numpy(video)
+        videos = [torch_video, torch_video]
+        videos_list = make_batched_videos(videos)
+        self.assertIsInstance(videos_list, list)
+        self.assertIsInstance(videos_list[0], torch.Tensor)
+        self.assertEqual(videos_list[0].shape, (8, 16, 32, 3))
+        self.assertTrue(np.array_equal(videos_list[0], video))
+
+    def test_resize(self):
+        video_processor = BaseVideoProcessorFast(model_init_kwargs=VideosKwargs)
+        video = get_random_video(16, 32, return_torch=True)
+
+        # Size can be an int or a tuple of ints.
+        size_dict = SizeDict(**get_size_dict((8, 8), param_name="size"))
+        resized_video = video_processor.resize(video, size=size_dict)
+        self.assertIsInstance(resized_video, torch.Tensor)
+        self.assertEqual(resized_video.shape, (8, 3, 8, 8))
+
+    def test_normalize(self):
+        video_processor = BaseVideoProcessorFast(model_init_kwargs=VideosKwargs)
+        array = torch.randn(4, 3, 16, 32)
+        mean = [0.1, 0.5, 0.9]
+        std = [0.2, 0.4, 0.6]
+
+        # mean and std can be passed as lists or NumPy arrays.
+        expected = (array - torch.tensor(mean)[:, None, None]) / torch.tensor(std)[:, None, None]
+        normalized_array = video_processor.normalize(array, mean, std)
+        torch.testing.assert_allclose(normalized_array, expected)
```

Comment:
_allclose is deprected

```suggestion
        torch.testing.assert_close(normalized_array, expected)
```